libsolidity/ast/AST.h

```diff
@@ -345,6 +345,31 @@ class VariableScope
 	std::vector<VariableDeclaration const*> m_localVariables;
 };
 
+/**
+ * The doxygen-style, structured documentation class that represents an AST node,
+ * which is created for other AST nodes that can receive such.
```

Comment:
This line is redundant because it is valid for all AST nodes apart from the topmost one :)

Reply:
ok!